synapse/rest/media/v1/preview_html.py

```diff
@@ -144,9 +143,7 @@ def decode_body(
     return etree.fromstring(body, parser)
 
 
-def parse_html_to_open_graph(
-    tree: "etree.Element", media_uri: str
-) -> Dict[str, Optional[str]]:
+def parse_html_to_open_graph(tree: "etree.Element") -> Dict[str, Optional[str]]:
     """
     Parse the HTML document into an Open Graph response.
 
```

Comment:
Just wondering if you want to remove the `media_url` parameter from the docstring below since it is being removed from the function signature?

Reply:
Good catch! Thanks! 👍